app/controllers/application_controller.rb

```diff
@@ -470,6 +471,15 @@ def fetch_translations
     WebTranslateIt.fetch_translations
   end
 
+  def check_http_auth
+    return true unless APP_CONFIG.use_http_auth
```

Comment:
If the `use_http_auth` is set using environment variable, the value is string.

Because only `nil` and `false` and falsey values, string such as `"false"` is considered truthy.

So before using the `APP_CONFIG` value, you should convert it to boolean, if it's a string.